### PR TITLE
Update to the latest swagger schema

### PR DESCRIPTION
**Pull request by kevinkirkup**
Update to the latest Swagger Schema.

### CODE REVIEW

File: docs/client/Account.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Description** | **string** |  | [optional] 
 **Href** | **string** |  | [optional] 
 **Id** | **string** |  | [optional] 
 **Name** | **string** |  | 
+**Description** | **string** |  | [optional] 
 **Parent** | [***Link**](Link.md) |  | [optional] 
 **SupportedConnectionGroups** | [**[]Link**](Link.md) |  | [optional]
```
Comment (Try75) [inline docs/client/Account.md]: Is there any way to get these properties to be output in the same order each run or are we causing this by moving things around?

Comment (Try75) [reply docs/client/Account.md]: It looks like it was alphabetically sorted and now is not.

File: docs/client/README.md [hunk 2]
```diff
@@ -66,18 +67,24 @@ Class | Method | HTTP request | Description
...
 *NetworksApi* | [**FindNetworks**](docs/NetworksApi.md#findnetworks) | **Get** /accounts/{accountId}/networks | List networks
-*NetworksApi* | [**GetConnections**](docs/NetworksApi.md#getconnections) | **Get** /networks/{networkId}/connections | List network connections
 *NetworksApi* | [**GetNetwork**](docs/NetworksApi.md#getnetwork) | **Get** /networks/{networkId} | Get network details
 *NetworksApi* | [**UpdateNetwork**](docs/NetworksApi.md#updatenetwork) | **Put** /networks/{networkId} | Update network
 *OptionsApi* | [**GetOptions**](docs/OptionsApi.md#getoptions) | **Get** /options | Get available options
+*PortsApi* | [**AddPort**](docs/PortsApi.md#addport) | **Post** /accounts/{accountId}/ports | Add new port
+*PortsApi* | [**DeleteNetwork1**](docs/PortsApi.md#deletenetwork1) | **Delete** /ports/{portId} | Delete port
+*PortsApi* | [**FindPorts**](docs/PortsApi.md#findports) | **Get** /accounts/{accountId}/ports | List ports
+*PortsApi* | [**GetPort**](docs/PortsApi.md#getport) | **Get** /ports/{portId} | Get port details
+*PortsApi* | [**UpdateNetwork1**](docs/PortsApi.md#updatenetwork1) | **Put** /ports/{portId} | Update port
```
Comment (Try75) [inline docs/client/README.md]: Did you have to do something to fix the names with a number at the end like this: UpdateNetwork1?

File: docs/client/Account.md (hunk 1 above)
Comment (kevinkirkup) [reply docs/client/Account.md]: Yeah, I think I had previously run `python -m json.tool` which sorts the schema alphabetically.
Let me generate the source. There were a could of bug fixes needed anyway.

File: docs/client/README.md (hunk 2 above)
Comment (kevinkirkup) [reply docs/client/README.md]: Yes, but the changes needed to be made in Kato to fix the generated Swagger document.

File: docs/client/Account.md (hunk 1 above)
Comment (kevinkirkup) [reply docs/client/Account.md]: Should be updated now.

File: docs/client/README.md (hunk 2 above)
Comment (kevinkirkup) [reply docs/client/README.md]: It also depends on the structure of the Kato Endpoint. If it is shared by 2 end-points, it will still show up this way, I'm pretty sure.

File: pureport/client/model_vpn_gateway.go [hunk 3]
```diff
@@ -21,7 +21,7 @@ type VpnGateway struct {
 	RemoteId           string         `json:"remoteId,omitempty"`
 	State              string         `json:"state,omitempty"`
 	Type_              string         `json:"type,omitempty"`
-	Auth               *PskAuthConfig `json:"auth,omitempty"`
+	Auth               *VpnAuthConfig `json:"auth,omitempty"`
```
Comment (kevinkirkup) [inline pureport/client/model_vpn_gateway.go]: This may need to be changed back to `PskAuthConfig`.

File: docs/client/README.md [hunk 4]
```diff
@@ -66,18 +67,24 @@ Class | Method | HTTP request | Description
...
 *SupportedConnectionsApi* | [**GetSupportedConnection**](docs/SupportedConnectionsApi.md#getsupportedconnection) | **Get** /supportedConnections/{supportedConnectionId} | Get supported connection details
-*SupportedConnectionsApi* | [**GetSupportedConnections**](docs/SupportedConnectionsApi.md#getsupportedconnections) | **Get** /accounts/{accountId}/supportedConnections | List supported connections
+*SupportedConnectionsApi* | [**GetSupportedConnections1**](docs/SupportedConnectionsApi.md#getsupportedconnections1) | **Get** /accounts/{accountId}/supportedConnections | List supported connections
```
Comment (Try75) [inline docs/client/README.md]: Referring to your PR to change Kato names, you missed this one.

Comment (kevinkirkup) [reply docs/client/README.md]: Yeah, not sure what is going on with this one. This is the only function for `GetSupportedConnections` and SwaggerGen in Kato is creating the number.
I can hand modify these for now though:
```
                "description": "",
                "operationId": "getSupportedConnections_1",
                "parameters": [
```

Comment (kevinkirkup) [reply docs/client/README.md]: Actually, it looks like there is an `AccountSupportedconnections` but is not included in the SwaggerAPI.

Comment (kevinkirkup) [reply docs/client/README.md]: Should be fix!

File: docs/client/NetworksApi.md [hunk 5]
```diff
@@ -1,55 +1,16 @@
...
-[**AddConnection**](NetworksApi.md#AddConnection) | **Post** /networks/{networkId}/connections | Add new connection
 [**AddNetwork**](NetworksApi.md#AddNetwork) | **Post** /accounts/{accountId}/networks | Add new network
 [**DeleteNetwork**](NetworksApi.md#DeleteNetwork) | **Delete** /networks/{networkId} | Delete network
 [**FindNetworks**](NetworksApi.md#FindNetworks) | **Get** /accounts/{accountId}/networks | List networks
-[**GetConnections**](NetworksApi.md#GetConnections) | **Get** /networks/{networkId}/connections | List network connections
 [**GetNetwork**](NetworksApi.md#GetNetwork) | **Get** /networks/{networkId} | Get network details
 [**UpdateNetwork**](NetworksApi.md#UpdateNetwork) | **Put** /networks/{networkId} | Update network
 
 
-# **AddConnection**
```
Comment (Try75) [inline docs/client/NetworksApi.md]: Why did AddConnection() and GetConnections() disappear from the Network resource?

Comment (kevinkirkup) [reply docs/client/NetworksApi.md]: Not sure about all of the Swagger voodoo here, but it looks like the `tags` from the Swagger Schema are used to assign methods to the generated API Services.
Not sure how or why Kato started tagging the swagger API differently, but if you look in the generated output, it was including the same API endpoint in both services:

https://github.com/pureport/pureport-sdk-go/blob/develop/pureport/client/api_connections.go#L402
https://github.com/pureport/pureport-sdk-go/blob/develop/pureport/client/api_networks.go#375

So basically just duplicates...

Comment (Try75) [reply docs/client/NetworksApi.md]: Interesting. Maybe something changed that stopped triggering that? Just wondered.
